pkgs/games/vinegar/default.nix

```diff
@@ -0,0 +1,60 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, wine
+, symlinkJoin
+, makeWrapper
+}:
+
+let
+  version = "1.4.5";
+
+  unwrapped = buildGoModule rec {
+    pname = "vinegar";
+
+    inherit version;
+
+    src = fetchFromGitHub {
+      owner = "vinegarhq";
+      repo = "vinegar";
+      rev = "v${version}";
+      hash = "sha256-FQ7ldW7cvAhHEcKLP5+IybGsRQdhM21/nTIoTKeU2Gs=";
+    };
+
+    vendorHash = "sha256-eRgmyl3y/E9InYD0q3q1jFhWpLFFBowS8WleeLKGWjM=";
+
+    makeFlags = [
+      "PREFIX=$(out)"
+    ];
+
+    buildPhase = ''
+      runHook preBuild
+      make $makeFlags
+      runHook postBuild
+    '';
+
+    installPhase = ''
+      runHook preInstall
+      make install $makeFlags
+      runHook postInstall
+    '';
+  };
+
+in
+symlinkJoin {
+  name = "vinegar";
+  paths = [ unwrapped ];
+  buildInputs = [ makeWrapper ];
+  meta = with lib; {
+    description = "An open-source, minimal, configurable, fast bootstrapper for running Roblox on Linux";
+    homepage = "https://github.com/vinegarhq/vinegar";
+    changelog = "https://github.com/vinegarhq/vinegar/releases/tag/v${version}";
+    mainProgram = "vinegar";
+    license = licenses.gpl3Only;
+    maintainers = with maintainers; [ ];
```

Comment:
Do you not want to maintain this?

Reply:
I am not qualified to maintain this :(

Reply:
You can put my name down if you'd like, just give me a moment to add myself to `lib.maintainers` 'cause I don't think I have yet.

Reply:
@YoshiRulz Did you add yourself to lib.maintainers?

Reply:
I did, yes. OP will need to rebase.